crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -1988,6 +2004,11 @@ impl Protobuf for BridgeTransfer {
                 "destination_chain_address",
             ));
         }
+        if destination_chain_address.as_bytes().len() > 0
+            && destination_chain_address.as_bytes().len() <= 256
```

Comment:
```suggestion
        if destination_chain_address.as_bytes().is_empty() || destination_chain_address.as_bytes().len() <= 256
```

ideally these will be two separate errors actually